pkgs/servers/monitoring/prometheus/nginx_exporter/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, goPackages, fetchFromGitHub }:
+
+goPackages.buildGoPackage rec {
+  name = "prometheus-nginx-exporter-${builtins.substring 0 7 rev}";
```

Comment:
My message was addressed to the author. There was no-one else.

I'm so fed up with GitHub.